### PR TITLE
feat(CollectionDetail): add sequence sorter

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/common/enums/url.ts [hunk 1]
```diff
@@ -9,3 +9,10 @@ export const URL_QS = {
 export const URL_ME_SETTINGS = {
   OPEN_SET_EMAIL_DIALOG: { key: 'dialog', value: 'set-email' },
 }
+
+export const URL_COLLECTION_DETAIL = {
+  SORTER_SEQUENCE: {
+    key: 'sorter-sequence',
+    value: { normal: 'normal', reverse: 'reverse' },
```
Comment (robertu7) [inline src/common/enums/url.ts:16]: `date-asc` and `date-dsc` for further extensibility (if we support more sorting methods)

File: src/views/User/CollectionDetail/CollectionArticles/index.tsx [hunk 2]
```diff
@@ -32,20 +33,47 @@ interface CollectionArticlesProps {
...
 
   const { articleList: articles, updatedAt } = collection
 
+  const changeSequence = (newSequence: SorterSequenceType) => {
+    setQuery(URL_COLLECTION_DETAIL.SORTER_SEQUENCE.key, newSequence)
+    setSequence(newSequence)
+    articles.edges = articles.edges?.reverse()
+  }
+
+  useEffect(() => {
```
Comment (robertu7) [inline src/views/User/CollectionDetail/CollectionArticles/index.tsx:70]: comments on Slack

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/common/enums/url.ts [hunk 3]
```diff
@@ -9,3 +9,10 @@ export const URL_QS = {
 export const URL_ME_SETTINGS = {
   OPEN_SET_EMAIL_DIALOG: { key: 'dialog', value: 'set-email' },
 }
+
+export const URL_COLLECTION_DETAIL = {
+  SORTER_SEQUENCE: {
+    key: 'sorter-sequence',
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/common/enums/url.ts:15]: Sequence is a linear order, one thing after another. There are other ways of ordering things that are not sequential

File: src/common/enums/url.ts (hunk 1 above)
Comment (pitb2022) [reply src/common/enums/url.ts:16]: Now that collection articles are not sorted by date, using date is not a good idea

**Review by robertu7 — APPROVED** [review]
LGTM 🍉

**Pull request merged**